sql/table_cache.cc

```diff
@@ -129,6 +129,26 @@ void Table_cache::free_all_unused_tables() {
   }
 }
 
+void Table_cache::free_old_unused_tables(time_point cutpoint) {
+  assert_owner();
+
+  if (!m_unused_tables) return;
+  std::vector<TABLE *> old_tables;
```

Comment:
typedef/using for a custom container.

Reply:
Are there any advantages of `using`? This type occurs only once. For me it doesn't look better:
```
  using table_vector = std::vector<TABLE *>;
  table_vector old_tables;
```